packages/lesswrong/components/community/Community.tsx

```diff
@@ -328,6 +348,11 @@ const Community = ({classes}: {
     )
   }
   
+  const handleToggleIncludeInactive = () => {
+    setIncludeInactive(!includeInactive)
+    history.replace({...location, search: `?includeInactive=${!includeInactive}`})
```

Comment:
Save the checkbox value in the URL so that it maintains state when you click back / forward (and the map respects the URL query params as well)